bitbots_vision/src/bitbots_vision/vision.py

```diff
@@ -321,10 +319,8 @@ def _dynamic_reconfigure_callback(self, config, level):
         self._ball_candidate_threshold = config['vision_ball_candidate_rating_threshold']
         self._ball_candidate_y_offset = config['vision_ball_candidate_field_boundary_y_offset']
 
-        self.debug_image = config['vision_debug_image']
-        self.debug_image_msg = config['vision_publish_debug_image']
-        self.debug = self.debug_image or self.debug_image_msg
-        if self.debug:
+        self.do_publish_debug_image = config['vision_publish_debug_image']
```

Comment:
the "do_"-part seems weird...

Reply:
resolved with caca46c0eb4cdbe93fae8d45f04c48d7d2bcf00b.